template/ui/build/script.javascript.js

```diff
@@ -5,13 +5,13 @@ const rollup = require('rollup')
 const uglify = require('uglify-es')
 const buble = require('@rollup/plugin-buble')
 const json = require('@rollup/plugin-json')
-const nodeResolve = require('@rollup/plugin-node-resolve')
+const resolve = require('@rollup/plugin-node-resolve')
```

Comment:
```suggestion
const { nodeResolve } = require('@rollup/plugin-node-resolve')
```
Doing it this way would preserve the old naming. Also, it makes more sense to use it like `nodeResolve` rather than `resolve.nodeResolve` IMO (just like before)